.github/workflows/labeler.yml

```diff
@@ -10,12 +10,12 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@main
+    - uses: actions/labeler@3e20f3d061896109eedd1ff7e9942a063b936e06 # main
```

Comment:
Is this effectively tip / there is no version?

Reply:
Yes, it was picking up the latest tip, so this locks to a specific point in time today Not sure why tip was used by the original workflow author, but happy to go back to the latest labelled release.